docs/source/format/Columnar.rst

```diff
@@ -790,7 +790,7 @@ type.
 **Example Layout: ``Struct<VarBinary, Int32>``**
 
 The layout for ``[{'joe', 1}, {null, 2}, null, {'mark', 4}]``, having
-child arrays ``['joe', null, 'alice', 'mark']`` and ``[1, 2, null, 4]``
+child arrays ``['joe', null, null, 'mark']`` and ``[1, 2, null, 4]``
```

Comment:
This is not wrong.
It shows that child arrays' elements aren't used for struct level null element.
(`child_arrays[0][2]` and `child_arrays[1][2]` are ignored when `struct[2]` is null.)

Reply:
> It shows that child arrays' elements aren't used for struct level null element.

To inform the user about this issue, we can offer an explanation accompanied by a relevant example. In this instance, the example is incorrect because 'alice' does not exist in the original dataset, preventing us from constructing the child array using the data.

Reply:
What is "the original dataset"?

BTW, did you read https://arrow.apache.org/docs/format/Columnar.html#struct-validity ?

Reply:
I have not read it😅. It's ok, I'll close this pr.